matchms/filtering/add_losses.py

```diff
@@ -20,7 +24,7 @@ def add_losses(spectrum_in: SpectrumType, loss_mz_from=0.0, loss_mz_to=1000.0) -
 
     spectrum = spectrum_in.clone()
 
-    precursor_mz = spectrum.get("precursor_mz")
+    precursor_mz = spectrum.get("precursor_mz", None)
     if precursor_mz:
         assert isinstance(precursor_mz, (float, int)), ("Expected 'precursor_mz' to be a scalar number.",
                                                         "Consider applying 'add_precursor_mz' filter first.")
```

Comment:
Yes, good point.
I can add those as `logger.error()` messages.

Reply:
Sorry, looked again and am less sure.
The asserts will stop the code execution anyway so usually there will at max be 1 logged message. It can be done, e.g. via 
```python
try:
    assert isinstance(precursor_mz, (float, int)), my_error_message
except AssertionError as err:
    logging.exception(err)
    raise
```
But that means a lot of additional lines of code...

Reply:
Maybe we can keep that as a new issue for future PRs.